doc/sources/_static/custom.css

```diff
@@ -121,3 +121,11 @@ div.quotation {
 
 a#wap_dns {display: none;}
 
+
+button#version-switcher-button {
+   background-color:  #2980b9;
+}
+
+div#version-switcher-dropdown {

```

Comment:
Question: if we have this line right here, do we still need to add a version selector in the `conf.py` file?
https://github.com/uxlfoundation/scikit-learn-intelex/blob/54aa18b56b853add58d8ea078efae36fb1209741/doc/sources/conf.py#L146

I understand that version selector plugin is meant for ReadTheDocs, so don't know how it plays with these.